asnets/asnets/scripts/run_learning.py

```diff
@@ -376,6 +377,8 @@ def main_inner(*,
         train_flags.extend(trained_problems)
         new_problem = problems.pop(0)
         train_flags.append(new_problem)
+        new_problem = problems.pop(0)
+        train_flags.append(new_problem)
```

Comment:
This code will break if there is only one problem left. Why do you need to add two problems at a time? (Same holds for code above, line 365).

Reply:
Because line 800 of supervised.py checks whether the two added problems have the same trainable weights. So at the beginning we need at least two problems. We added two problems at a time after the training time was reduced to 24 hours in the hope that the planner could learn from more problems. It looks like this strategy has not been committed before. Can we still make this change now?

Reply:
Ensuring that there are two problems at the start is fine, since this sounds like a bugfix. Adding two problems at a time later sounds like an optimization, not a bugfix, so let's remove this change.

For the initialization, please add something like 
```
if len(problems) < 2:
    sys.exit("Error: we need at least two tasks for training.")
```

Reply:
Have you had a chance to look into this? We're about to start experiments for our journal submission and it would be great to include your fixes.

Reply:
Sorry for not getting back to you sooner. I have been a bit busy with another project recently. I have fixed the code by removing these two lines. By the way, it looks like the paper on the website is not yet updated. Could you please have a look? Thanks!

Reply:
Thanks for making the change! Unfortunately, the build is broken (see the GitHub Action log). I suspect that you either need to install an older version of pip (pinning the version is recommended anyway), or add a file asnets/pyproject.toml with the contents:

```
[build-system]
requires = ["setuptools"]
build-backend = "setuptools.build_meta"
``` 

PS: I uploaded your new planner abstract, thanks for the reminder!

Reply:
Thank you for the suggestions! It looks like the build is successful now :)

Reply:
Yes, looks good, thanks! Merged :-)